Fixed a sentence (#25536)
mdn/content: files/en-us/web/css/css_grid_layout/relationship_of_grid_layout/index.md

{{ EmbedLiveSample('A_flexible_number_of_tracks', '500', '170') }}

> So not only do we now have the ability to create a grid with a dynamic number of flexible tracks, we can also have the items laid out on the grid aligned by rows and columns.

Reviewer: ```suggestion
With grid layout, we can create a grid with a dynamic number of flexible tracks and have the items laid out on the grid aligned by rows and columns.
```

{{ EmbedLiveSample('A_flexible_number_of_tracks', '500', '170') }}

> So not only do we now have the ability to create a grid with a dynamic number of flexible tracks, we can also have the items laid out 

Reviewer: The original was written when this was a new feature. I suggest we get rid of the "now" new language.
